src/routes/SafeAppLandingPage/components/UseYourSafe.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState, ReactElement } from 'react'
+import { Link } from 'react-router-dom'
+import { Text, Button } from '@gnosis.pm/safe-react-components'
+import styled from 'styled-components'
+
+import SafeAddressSelector from 'src/components/SafeAddressSelector/SafeAddressSelector'
+import { getChainById } from 'src/config'
+import { AddressBookEntry } from 'src/logic/addressBook/model/addressBook'
+import { generateSafeRoute, SAFE_ROUTES } from 'src/routes/routes'
+
+type UseYourSafeTypes = {
+  safeAppUrl: string
+  defaultSafe: AddressBookEntry
+  safes: AddressBookEntry[]
+}
+
+const UseYourSafe = ({ safes, safeAppUrl, defaultSafe }: UseYourSafeTypes): ReactElement => {
```

Comment:
This component is displayed when compatible Safes are available (from service and localstorage):

![Captura de pantalla 2022-05-04 a las 14 51 26](https://user-images.githubusercontent.com/26763673/166684767-6f3eebd7-a169-48b3-8efe-77f9f2a87b1f.png)